src/main/java/frc/robot/commands/backwardAuto.java

```diff
@@ -11,6 +11,7 @@
 public class backwardAuto extends CommandBase {
   /** Creates a new backwardAuto. */
```

Comment:
When doing the rewrite, lets make sure to organize commands better into their own folders. ie `commands/auto/backwardAuto.java`